### PR TITLE
Bluetooth: controller: llcp: fix phy regression errors on EBQ

### PR DESCRIPTION
**Pull request by kruithofa**
Some of the peripheral PHY EBQ tests started failing due to 
regression error:
transmission of data was paused but not resumed.
This is fixed by this commit

Affected tests on EBQ:
LL/CON/PER/BV-49-C
LL/CON/PER/BV-50-C
LL/CON/PER/BV-52-C
LL/CON/PER/BV-53-C
LL/CON/PER/BV-54-C
LL/CON/PER/BV-55-C
LL/CON/PER/BV-56-C
LL/CON/PER/BV-58-C

depends on PR46394

Signed-off-by: Andries Kruithof <andries.kruithof@nordicsemi.no>

### CODE REVIEW

**Review by cvinayak — DISMISSED** [review]

File: subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c [hunk 1]
```diff
@@ -563,6 +563,7 @@ static void lp_pu_st_wait_tx_ack_phy_req(struct ll_conn *conn, struct proc_ctx *
 				conn, pu_select_phy_timing_restrict(conn, ctx->data.pu.tx));
 			ctx->state = LP_PU_STATE_WAIT_RX_PHY_UPDATE_IND;
 			ctx->rx_opcode = PDU_DATA_LLCTRL_TYPE_PHY_UPD_IND;
+			llcp_tx_resume_data(conn, LLCP_TX_QUEUE_PAUSE_DATA_PHY_UPDATE);
```
Comment (thoh-ot) [inline subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c:566]: Why doesn't the `llcp_tx_resume_data(conn, LLCP_TX_QUEUE_PAUSE_DATA_PHY_UPDATE);` in line 573 work?

Comment (kruithofa) [reply subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c:566]: That line works for peripheral, but causes problems for Central EBQ conformance tests, and is removed in PR #46394 
Waiting for PR46394 to be approved and merged.

Comment (kruithofa) [reply subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c:566]: Or actually, I'll move the change into PR46394, and close this one

**Comment by kruithofa** [comment]
Changes are part of #46394

**Pull request closed without merging**